src/sentry/auth/providers/saml2/provider.py

```diff
@@ -323,6 +323,13 @@ def refresh_identity(self, auth_identity):
         return
 
 
+class SCIMMixin:
+    def can_use_scim(self, organization, user):
+        if features.has("organizations:sso-scim", organization, actor=user):
```

Comment:
Long term it will probably be based solely on the type of AuthProvider. I could see it where we have flags for providers we want to add to scim support but want to trial it with early adopters first as well.